pym/bob/cmds/build.py

```diff
@@ -333,6 +333,8 @@ def _generateAudit(self, step, depth, resultHash):
         audit = Audit.create(step.getVariantId(), self._getBuildId(step, depth), resultHash)
         audit.addDefine("bob", BOB_VERSION)
         audit.addDefine("recipe", step.getPackage().getRecipe().getName())
+        for var,val in step.getPackage().getMetaVars().items():
+            audit.addDefine(var, val)
```

Comment:
Hmm, this may collide with the other "defines". Either we rename existing variables to something that cannot collide (e.g. `.bob`, `.recipe`) or put meta variables in a new namespace (e.g. metaEnvironment). Thoughts?